content/en/get-started/_index.md

```diff
@@ -57,6 +57,13 @@ The easiest way to install the LocalStack CLI is via `pip`:
 $ python3 -m pip install localstack
 {{< / command >}}
 
+Note: To download specific version of localstack, provide version to the above command.
+For example, to install localstack version v1.1.0 use,
```

Comment:
Can we instead pick up `<version>` instead of providing an arbitrary version? We would release 1.2.0 soon and it would be outdated

Reply:
@HarshCasper did not fully understood the required change.
As this is an example, version in example can be any valid version.

Please share your thoughts on this.
And thank you for all previous inputs.

Reply:
Hi @Sagar2366 — I believed it can be rephrased in the following manner:

```
To download a specific version of LocalStack, check out our [release page](https://github.com/localstack/localstack) and download it in the following manner:

{{< command >}} 
$ python3 -m pip install localstack==<version>
{{< / command >}}

Here `<version>` depicts the particular LocalStack version that you would like to download and use.
```

I don't prefer using arbitrary value-based examples since they are subject to change all the time. I hope it makes sense now.

Reply:
I thought we're expecting to pull latest version available and show it.
Thank you for clarifying.